Formula/opencv.rb

```diff
@@ -0,0 +1,157 @@
+class Opencv < Formula
+  desc "Open source computer vision library, version 3"
+  homepage "http://opencv.org/"
```

Comment:
actually does need revision 1 if we're going to do the formula rename from opencv3